src/node.cpp

```diff
@@ -325,6 +332,26 @@ Node* Node::unserialize(MegaClient* client, string* d, node_vector* dp)
 
     ptr = n->attrs.unserialize(ptr);
 
+    PublicLink *plink = NULL;
+    if (version_flag >= 1)
+    {
```

Comment:
We could save memory by directly having the isExported flag in the reserved bytes above, instead of the version_flag.